arcgis10_mapping_tools/MapAction/MapAction/PageLayoutProperties.cs

```diff
@@ -473,18 +473,7 @@ public static string getMxdTitle(IApplication pApp)
                 {

```

Comment:
a bit of old code outside this review, I know, but... "if boolMethod() == true"... urgh!